utilities/plotting/plotRefSLD.m

```diff
@@ -7,9 +7,9 @@ function plotRefSLD(problem, result)
     data.reflectivity = result.reflectivity;
     data.shiftedData = result.shiftedData;
     data.sldProfiles = result.sldProfiles;
-    data.allLayers = result.allLayers;
+    data.resampledLayers = result.resampledLayers;
     data.dataPresent = problemStruct.dataPresent;
-    data.ssubs = result.contrastParams.ssubs;
+    data.subRoughs = result.contrastParams.subRoughs;
```

Comment:
I think there is a discrepancy here. You rename ssubs to subRoughs here but completely removed ssubs in triggerEvents.m

Reply:
Ahh. Yeah, they should have been renamed. On it now!